packages/node_modules/@node-red/editor-client/locales/de/editor.json

```
@@ -1156,7 +1166,8 @@
     "tourGuide": {
         "takeATour": "Tour starten",
         "start": "Start",
-        "next": "Nächste"
+        "next": "Nächste",
+        "welcomeTours": "Willkommens-Touren"
```

```suggestion
        "welcomeTours": "Welcome Touren"
```
This is a mixture of two languages - that I'm convinced German users will understand.

---

Personally, I don't like the mixture of the two languages...
"Welcome Tours" sounds strange, I would be for either "Willkommens-Touren" (as suggested) or "Welcome Tours" (original).

---

I'm fine with keeping this "Welcome Tours". Maybe we find a better option sometime later...